mirrulations-client/src/mirrclient/client.py

```diff
@@ -54,9 +62,27 @@ def send_job_results(self, job_id, job_result):
         # print(dumps(data))
         assure_request(requests.put, endpoint, json=dumps(data), params=params)
 
+    def send_attachment_results(self, job_id, job_result): # will be similar to send_job_results. Need to collaborate with work server people...
+        endpoint = f'{self.url}/put_results'
+        if 'errors' in job_result:
+            data = {
+                    'job_id': job_id,
+                    'results': job_result
+                    }
+        else:
+            data = {'directory': get_output_path(job_result),
+                    'job_id': job_id,
+                    'results': job_result}
+            data['results']['attachment_text'] = ['foo']
+            data['results']['type'] = ['attachment']
```

Comment:
Line 76 is perfect, should stay as is.

Line 77 shouldn't have 'attachment' in a list, it should be 
```
data['results']['type'] = 'attachment'
```